README.md

```diff
@@ -66,7 +71,7 @@ docker run -ti \
     # Adjust the container workspace to the newly mounted project.
     -w "/go/src/${project_path}" \
     # Run a command.
-    asecurityteam/sdcli:v1 go test
+    asecurityteam/sdcli:latest go test
```

Comment:
Question: why are you changing this to latest?

Reply:
My thought was that eventually we might not be using the v1 tag for our sdcli image, so it might make more since just to pull the latest image by default.

Reply:
So I'm rolling back the change, as I noticed that having sdcli always pull from latest was causing a conflict on my system where it would only use the image hosted on docker hub and not my own personal image.